src/compas_occ/brep/brep.py

```diff
@@ -1245,6 +1266,25 @@ def contours(
         """
         raise NotImplementedError
 
+    def trim(self, plane: Union[compas.geometry.Plane, compas.geometry.Frame]):
+        """Trim a BRep with a plane.
+
+        Same as BRep.slice but performed in-place.
```

Comment:
slice does something else. it slices through the brep with a plane and returns the geometry of the slice...

Reply:
i would propose:
* slice: slice through and return the geometry of the slice
* split: split into two parts
* trim: remove the parts on the other side of the trim

Reply:
* slice, sliced
* split, splitted (?)
* trim, trimmed

Reply:
fair enough